draft-scenarios/clash/gold_rush.tex

```diff
@@ -0,0 +1,145 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Solo/Clash Scenario}{Gold Rush}{\images/estates.png}
```

Comment:
```suggestion
\addscenariosection{1}{Solo/Clash/Alliance Scenario}{Gold Rush}{\images/estates.png}
```
I guess it's that one too?

Reply:
Yes, however Alliance mode is still untested.

Reply:
No problem, this is what the Draft Book is for. Those scenarios don't need to be playtested. Mission Book has this requirement.